CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 # Changelog
+## v1.4.2
+#### 4844 related
+[internal/ethapi: add support for blobs in eth_fillTransaction (#28839)](https://github.com/buddh0/bsc/commit/ac5aa672d3b85a1f74667a65a15398f072aa0b2a)
```

Comment:
also need to replace `buddh0` with `bnb-chain`

Reply:
updated